common/src/main/java/io/homeassistant/companion/android/common/data/keychain/KeyChainRepositoryImpl.kt

```diff
@@ -0,0 +1,62 @@
+package io.homeassistant.companion.android.common.data.keychain
+
+import android.content.Context
+import android.security.KeyChain
+import io.homeassistant.companion.android.common.data.prefs.PrefsRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
```

Comment:
You can easily detect these errors by running `gradlew ktlintCheck` inside Android Studio too.

Once this branch no longer has a merge conflict we will be able to run Github actions which will actually fail for things like this during the lint check. It will also generate a debug APK for others to test with.